CONTRIBUTING.md

```diff
@@ -38,6 +38,13 @@ For example, for running tests in `multi`, `uuid` and `main` packages, call
 make test-multi test-uuid test-main
 ```
 
+To check if the current changes will pass the linter in CI, install
+golnagci-lint from [sources](https://golangci-lint.run/usage/install/) 
+and run it with next flags:
+```bash
+golangci-lint run -E gofmt -D errcheck,structcheck,unused test_helpers 
```

Comment:
Test helpers?

Reply:
Running it wothout a file name will check all files. It is the most useful option, as for me.

Reply:
Fixed.

Reply:
Fixed.